templates/conll2003/templates.yaml

```diff
@@ -0,0 +1,118 @@
+dataset: conll2003
+templates:
+  52b4e787-634b-4334-b490-bdb38e0082ab: !Template
+    id: 52b4e787-634b-4334-b490-bdb38e0082ab
+    jinja: "{% set _chunk_label_dict = ({\n0:\"O\",\n1:\"B-ADJP\",\n2:\"I-ADJP\",\n\
+      3:\"B-ADVP\",\n4:\"I-ADVP\",\n5:\"B-CONJP\",\n6:\"I-CONJP\",\n7:\"B-INTJ\",\n\
+      8:\"I-INTJ\",\n9:\"B-LST\",\n10:\"I-LST\",\n11:\"B-NP\",\n12:\"I-NP\",\n13:\"\
+      B-PP\",\n14:\"I-PP\",\n15:\"B-PRT\",\n16:\"I-PRT\",\n17:\"B-SBAR\",\n18:\"I-SBAR\"\
+      ,\n19:\"B-UCP\",\n20:\"I-UCP\",\n21:\"B-VP\",\n22:\"I-VP\"\n}) %}\n{% set _ner_label_dict\
+      \ = ({\n0:\"O\",\n1:\"B-PER\",\n2:\"I-PER\",\n3:\"B-ORG\",\n4:\"I-ORG\",\n5:\"\
+      B-LOC\",\n6:\"I-LOC\",\n7:\"B-MISC\",\n8:\"I-MISC\"\n}) %}\n{% set _pos_label_dict\
+      \ = ({\n0:'\"',\n1:\"''\",\n2:\"#\",\n3:\"\\$\",\n4:\"(\",\n5:\")\",\n6:\",\"\
+      ,\n7:\".\",\n8:\":\",\n9:\"\\`\\`\",\n10:\"CC\",\n11:\"CD\",\n12:\"DT\",\n13:\"\
+      EX\",\n14:\"FW\",\n15:\"IN\",\n16:\"JJ\",\n17:\"JJR\",\n18:\"JJS\",\n19:\"LS\"\
+      ,\n20:\"MD\",\n21:\"NN\",\n22:\"NNP\",\n23:\"NNPS\",\n24:\"NNS\",\n25:\"NN|SYM\"\
+      ,\n26:\"PDT\",\n27:\"POS\",\n28:\"PRP\",\n29:\"PRP\\$\",\n30:\"RB\",\n31:\"\
+      RBR\",\n32:\"RBS\",\n33:\"RP\",\n34:\"SYM\",\n35:\"TO\",\n36:\"UH\",\n37:\"\
+      VB\",\n38:\"VBD\",\n39:\"VBG\",\n40:\"VBN\",\n41:\"VBP\",\n42:\"VBZ\",\n43:\"\
+      WDT\",\n44:\"WP\",\n45:\"WP$\",\n46:\"WRB\"\n}) %}\n{% set _task = [\"named\
+      \ entities\", \"chunk tag\", \"parts of speech\"] | choice %}\n{% set _label_dict\
```

Comment:
can you separate all the template per task ("named entities", "chunk tag", "parts of speech")? it's very hard to read or review right now...

Reply:
Ok. Separating.